components/organisms/GitHubTeamImportDialog/github-team-import-dialog.tsx

```diff
@@ -0,0 +1,131 @@
+  open,
+  handleClose,
+  handleImport,
+  loading,
+  username,
+}: GitHubTeamImportDialogProps) => {
+  const [follow, setFollow] = useState(false);
+  const [selectedOrg, setSelectedOrg] = useState("");
+  const [teamSlug, setTeamSlug] = useState("");
+  const [teams, setTeams] = useState<GhOrgTeam[]>([]);
+
+  const { data: userOrgs } = useUserOrganizations(username);
+  const { toast } = useToast();
+
+  async function loadTeams(org: string) {
+    const response = await fetchGithubOrgTeams(org);
+
+    setTeams([]);
+    setTeamSlug("");
+
+    if (response.isError) {
+      toast({
+        description: "There was error loading teams. Check your organization permissions",
+        variant: "warning",
+      });
+
+      return;
+    }
+
+    setTeams(response.data);
+  }
+
+  useEffect(() => {
+    if (selectedOrg) {
+      loadTeams(selectedOrg);
+    }
+  }, [selectedOrg]);
+
+  return (
+    <Dialog open={open}>
+      <DialogContent>
+        <div className="flex flex-col max-w-xs gap-6 w-max">
+          <div className="flex flex-col gap-2">
+            <span className="flex items-center justify-center p-2 bg-black rounded-full w-max">
+              <span className="flex items-center justify-center w-10 h-10 bg-white rounded-full">
+                <FiGithub size={24} className="text-blue-12" />
+              </span>
+            </span>
+            <Title level={3} className="text-lg">
+              Import a GitHub Team
+            </Title>
+            <Text className="leading-tight text-light-slate-9">
+              We will import contributors from your team on GitHub to create your new list.
+            </Text>
+            <div className="flex pt-2">
+              Organization
+              <SingleSelect
+                options={userOrgs.map((userOrg) => ({
+                  label: userOrg.organization_user.login,
+                  value: `${userOrg.organization_user.login}`,
+                }))}
+                position="popper"
+                value={selectedOrg}
+                placeholder="Select an organization"
+                onValueChange={(value) => setSelectedOrg(value)}
+              />
+            </div>
+            <div className="flex pt-2">
+              Team
+              <SingleSelect
+                options={teams.map((team) => ({ label: team.name, value: team.slug }))}
+                position="popper"
+                value={teamSlug}
+                placeholder="Select a team"
+                onValueChange={(value) => setTeamSlug(value)}
+              />
+            </div>
+            <div className="flex pt-2">
+              <ToggleSwitch
+                name="followImported"
+                classNames="mr-2"
+                checked={follow}
+                handleToggle={() => setFollow((toFollow) => !toFollow)}
+              />
+              <Text className="leading-tight text-light-slate-9">Follow everyone imported.</Text>
+            </div>
```

Comment:
refactor (blocking): form elements need a label.

```suggestion
            <label className="flex mt-4 gap-4 items-center">
              <span>Organization</span>
              <SingleSelect
                options={userOrgs.map((userOrg) => ({
                  label: userOrg.organization_user.login,
                  value: `${userOrg.organization_user.login}`,
                }))}
                position="popper"
                value={selectedOrg}
                placeholder="Select an organization"
                onValueChange={(value) => setSelectedOrg(value)}
              />
            </label>
            <label className="flex gap-4 items-center">
              <span>Team</span>
              <SingleSelect
                options={teams.map((team) => ({ label: team.name, value: team.slug }))}
                position="popper"
                value={teamSlug}
                placeholder="Select a team"
                onValueChange={(value) => setTeamSlug(value)}
              />
            </label>
            <label className="flex">
              <ToggleSwitch
                name="followImported"
                classNames="mr-2"
                checked={follow}
                handleToggle={() => setFollow((toFollow) => !toFollow)}
              />
              <Text className="leading-tight text-light-slate-9">Follow everyone imported.</Text>
            </label>
```

**Before**

![CleanShot 2023-10-25 at 17 17 08](https://github.com/open-sauced/app/assets/833231/ad8184ea-9511-4358-8a59-9a013b068df1)

**After**

![CleanShot 2023-10-25 at 17 16 40](https://github.com/open-sauced/app/assets/833231/be9865ca-6dd5-4f25-851d-09b496eaaee3)

Reply:
A bigger discussion @isabensusan, but we'll need to work on colour contrast issues with the text and backgrounds throughout the app.